src/Userenv.Desktop/Userenv.cs

```diff
@@ -32,7 +32,7 @@ public static partial class Userenv
         /// </returns>
         /// <remarks>
         /// To free the buffer when you have finished with the environment block, call the DestroyEnvironmentBlock function.
-        /// If the environment block is passed to <see cref="CreateProcessAsUser(IntPtr, string, string, SECURITY_ATTRIBUTES*, SECURITY_ATTRIBUTES*, bool, CreateProcessFlags, void*, string, ref STARTUPINFO, out PROCESS_INFORMATION)"/>, you must also specify the CREATE_UNICODE_ENVIRONMENT flag. After CreateProcessAsUser has returned, the new process has a copy of the environment block, and <see cref="DestroyEnvironmentBlock(char*)"/> can be safely called.
+        /// If the environment block is passed to CreateProcessAsUsers, you must also specify the CREATE_UNICODE_ENVIRONMENT flag. After CreateProcessAsUser has returned, the new process has a copy of the environment block, and <see cref="DestroyEnvironmentBlock(char*)"/> can be safely called.
```

Comment:
Please restore the `<see cref` tag, adding parameter types to resolve the compiler warning if necessary.